### PR TITLE
[wasm] Mark System.ComponentModel APIs as unsupported on Browser

### PR DESCRIPTION
**Pull request by mdh1418**
Contributes to #41087 

```
"M:System.Security.Authentication.ExtendedProtection.ExtendedProtectionPolicyTypeConverter.ConvertTo(System.ComponentModel.ITypeDescriptorContext,System.Globalization.CultureInfo,System.Object,System.Type)",System.Security.Authentication.ExtendedProtection,ExtendedProtectionPolicyTypeConverter,"ConvertTo(ITypeDescriptorContext, CultureInfo, Object, Type)",1
"M:System.ComponentModel.TypeDescriptor.CreateInstance(System.IServiceProvider,System.Type,System.Type[],System.Object[])",System.ComponentModel,TypeDescriptor,"CreateInstance(IServiceProvider, Type, Type[], Object[])",3
"M:System.ComponentModel.TypeDescriptionProvider.CreateInstance(System.IServiceProvider,System.Type,System.Type[],System.Object[])",System.ComponentModel,TypeDescriptionProvider,"CreateInstance(IServiceProvider, Type, Type[], Object[])",2
"M:System.ComponentModel.LicenseManager.CreateWithContext(System.Type,System.ComponentModel.LicenseContext,System.Object[])",System.ComponentModel,LicenseManager,"CreateWithContext(Type, LicenseContext, Object[])",2
"M:System.ComponentModel.LicenseManager.CreateWithContext(System.Type,System.ComponentModel.LicenseContext)",System.ComponentModel,LicenseManager,"CreateWithContext(Type, LicenseContext)",3
M:System.ComponentModel.MaskedTextProvider.Clone,System.ComponentModel,MaskedTextProvider,Clone(),2
```

### CODE REVIEW

File: src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using System.Diagnostics;
 using System.Reflection;
 using System.Threading;
+using System.Versioning;
```
Comment (MaximLipnin) [inline src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs:9]:
```suggestion
using System.Runtime.Versioning;
```

File: src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/MaskedTextProvider.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics;
 using System.Globalization;
 using System.Text;
+using System.Versioning;
```
Comment (MaximLipnin) [inline src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/MaskedTextProvider.cs:10]:
```suggestion
using System.Runtime.Versioning;
```

File: src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptionProvider.cs [hunk 3]
```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections;
+using System.Versioning;
```
Comment (MaximLipnin) [inline src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptionProvider.cs:5]:
```suggestion
using System.Runtime.Versioning;
```

File: src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs [hunk 4]
```diff
@@ -9,6 +9,7 @@
 using System.Globalization;
 using System.Reflection;
 using System.Threading;
+using System.Versioning;
```
Comment (MaximLipnin) [inline src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs:12]:
```suggestion
using System.Runtime.Versioning;
```

File: src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs (hunk 1 above)
Comment (mdh1418) [reply src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs]: Oops, thanks!

File: src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptionProvider.cs (hunk 3 above)
Comment (mdh1418) [reply src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptionProvider.cs]: Thanks!

File: src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs (hunk 4 above)
Comment (mdh1418) [reply src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs]: Thanks :)

**Review by steveisok — APPROVED** [review]

**Review by safern — APPROVED** [review]
Out of curiosity how and why did we decide that these were the not supported ones?

**Comment by steveisok** [comment]
@safern There's a scanner that will surface PNSE.  We're trying to use what we can from that list.

**Pull request merged**